packages/storybook8/stories/Examples/CustomButtonInjection/CustomButtonInjection.story.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { CallComposite, CustomCallControlButtonCallback } from '@azure/communication-react';
+import { _MockCallAdapter } from '@internal/react-composites';
+import React from 'react';
+
+//boiler plate for testing
+const maxCustomButtonsForInjection: CustomCallControlButtonCallback[] = [
+  () => ({
+    placement: 'primary',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    showLabel: tempShowLabel,
+    disabled: tempDisabled
+  }),
+  () => ({
+    placement: 'primary',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    showLabel: tempShowLabel,
+    disabled: tempDisabled
+  }),
+  () => ({
+    placement: 'primary',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    showLabel: tempShowLabel,
+    disabled: tempDisabled
+  }),
+  () => ({
+    placement: 'secondary',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    showLabel: tempShowLabel,
+    disabled: tempDisabled
+  }),
+  () => ({
+    placement: 'secondary',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    showLabel: tempShowLabel,
+    disabled: tempDisabled
+  }),
+  () => ({
+    placement: 'overflow',
+    iconName: 'DefaultCustomButton',
+    strings: {
+      label: 'Custom',
+      tooltipContent: 'Custom'
+    },
+    disabled: tempDisabled
+  })
+];
+
+let tempShowLabel: boolean;
+let tempDisabled = false;
+
+const CustomButtonInjectionStory = (args: any): JSX.Element => {
+  const adapter = new _MockCallAdapter({});
+  tempShowLabel = args.showButtonLabel !== 'undefined' ? args.showButtonLabel : undefined;
```

Comment:
I know this is copied from the original. But just as a callout it does seem odd that the API is a boolean allows true false, and we allow `undefined` IMO this should just be a toggle and be true / false

Reply:
Optional prop do optional prop things. 
Optional be optional.